test/puppetlabs/trapperkeeper/services/status/status_service_test.clj

```diff
@@ -280,7 +286,14 @@
           [foo-service baz-service fail-service]
           (let [resp (http-client/get "http://localhost:8180/status/v1/simple")]
             (is (= 503 (:status resp)))
-            (is (= "error" (slurp (:body resp)))))))
+            (is (= "unknown" (slurp (:body resp)))))))
```

Comment:
@aperiodic no need to apologize, and my mistake for letting your comment slip by me somehow. If I understand your proposal, I think my only concern is that the precedence between starting and stopping becomes confusing when someone hits the /simple endpoint while one service is starting and another is stopping. That state really does feel more like "unknown" to me. I do agree that my implementation here should be changed back to reflect :error if any of the services return that state. Any additional thoughts on starting/stopping?